Scoop/Controller/TripExtraDetailsViewController.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  TripExtraDetailsViewController.swift
+//  Scoop
+//
+//  Created by Elvis Marcelo on 3/18/22.
+//
+
+import UIKit
+
+class TripExtraDetailsViewController: UIViewController {
+    
+    private let travelersLabel = UILabel()
+    private let toLabel = UILabel()
+    private let count1Field = UITextField()
+    private let count2Field = UITextField()
+    
+    private let dateLabel = UILabel()
+    private let dateField = UITextField()
+    
+    private let timeLabel = UILabel()
+    private let timeField = UITextField()
+    
+    private let detailsLabel = UILabel()
+    private let detailsField = UITextField()
+
+
+    override func viewDidLoad() {
+        
+
+        
+        self.navigationItem.title = "Trip Details"
+        view.backgroundColor = .white
+        
+        travelersLabel.text = "Number of Travelers"
+        travelersLabel.textColor = .black
+        view.addSubview(travelersLabel)
+        
+        let peopleImage = UIImage(systemName: "person.2")
```

Comment:
You seem to create the UIImage in line with the ImageView for all the other the image views. If that's the style you are going for, you might want to change this to fit.